server/views/index.hbs

```diff
@@ -1,3 +1,24 @@
- <h1>{{title}}</h1>
-<p>Welcome to {{title}}</p>
+<table class="table table-bordered table-danger">
```

Comment:
this is very similar to the table in the admin-list-categories. Can you DRY the code? maybe use handlebars partials?